src/main/java/ThirdStep/Play.java

```diff
@@ -0,0 +1,24 @@
+package ThirdStep;
+
+import java.util.List;
+import java.util.Random;
+
+public class Play {
+    public static void racing(List<Car> cars, int attempts) {
+        if (attempts == 0) {
+            return;
+        }
+
+        cars.forEach(car -> {
+            car.move(Play.CanMoveForward());
+            car.printLocation();
+        });
+
+        System.out.println();
+        Play.racing(cars, --attempts);
+    }
+
+    private static boolean CanMoveForward() {
```

Comment:
네이밍 규칙 한번 확인 부탁 드립니다 🙇 
```suggestion
    private static boolean canMoveForward() {
```